notebook/notebook/notebook.py

```diff
@@ -34,40 +65,37 @@
     handlers=[fh, ch],
     level=logging.INFO)
 
-if 'BATCH_USE_KUBE_CONFIG' in os.environ:
-    kube.config.load_kube_config()
-else:
-    kube.config.load_incluster_config()
+log.info(f'AUTH_GATEWAY: {AUTH_GATEWAY}')
```

Comment:
for future PRs, moving the last two print statements as a part of a PR that makes a lot of other changes hurts diff readability. It's much less clear that the last two lines have not changed.